### ISSUE TITLE
About definition in Glossary: Functional need in Explainer for WCAG 3.0

### ISSUE THREAD

**Issue by spanchang** [w3c/wcag3#314]
Refer to Glossary section in https://www.w3.org/TR/wcag-3.0-explainer/

Functional need-
A statement that describes a specific gap in one’s ability, or a specific mismatch between ability and the designed environment or context.
Comment: i. Expressing it in a positive  fashion without "gap", "mismatch" might be more useful. 
ii. Is functional need really the gap or mismatch? The need is the whole of the ability. The gap is what AT provides accommodation for. 
Alternative definition for consideration:
Functional need means skills or abilities required in order to reach specified outcomes while using digital content in the environment or context it is designed for.

**Comment by airbr** [comment]
Good commentary, not sure what the definition should be but good comment to my view

**Comment by spanchang** [comment]
Related issue:
https://github.com/w3c/wcag3/issues/416
